README.md

```diff
@@ -68,35 +68,17 @@ of the features in the library.
 You can parse an HTML document using the following workflow:
 
 ```python
-from unstructured.documents.html import HTMLDocument
-
-doc = HTMLDocument.from_file("example-docs/example-10k.html")
-print(doc.pages[2])
-```
-
-The output of this will be the following:
-
-```
-SPECIAL NOTE REGARDING FORWARD-LOOKING STATEMENTS
+from unstructured.partition.html import partition_html
 
-This report contains statements that do not relate to historical or current facts but are “forward-looking” statements. These statements relate to analyses and other information based on forecasts of future results and estimates of amounts not yet determinable. These statements may also relate to future events or trends, our future prospects and proposed new products, services, developments or business strategies, among other things. These statements can generally (although not always) be identified by their use of terms and phrases such as anticipate, appear, believe, could, would, estimate, expect, indicate, intent, may, plan, predict, project, pursue, will continue and other similar terms and phrases, as well as the use of the future tense.
-
-Actual results could differ materially from those expressed or implied in our forward-looking statements. Our future financial condition and results of operations, as well as any forward-looking statements, are subject to change and to inherent known and unknown risks and uncertainties. You should not assume at any point in the future that the forward-looking statements in this report are still valid. We do not intend, and undertake no obligation, to update our forward-looking statements to reflect future events or circumstances.
-```
-
-If you then run:
-
-```python
-doc.pages[2].elements
+elements = partition_html("example-docs/example-10k.html")
```

Comment:
I slightly prefer showing the actual text contents of the elements here since someone approaching this for the first time will see the beginning and end points in terms they understand without knowing anything about the `Title` and `NarrativeText` objects. Not a blocker though.

Reply:
Added the text output back into the readme